server/src/main/java/dev/slimevr/Main.kt

```diff
@@ -49,6 +47,16 @@ fun main(args: Array<String>) {
 		println("SlimeVR Server $VERSION")
 		exitProcess(0)
 	}
+
+	if (cmd.args.isEmpty()) {
+		System.err.println("No command specified, expected 'run'")
+		exitProcess(1)
+	}
+	if (cmd.args[0] != "run") {
```

Comment:
This could be case-insensitive
```suggestion
	if (!cmd.args[0].equalsIgnoreCase("run")) {
```